package.json

```diff
@@ -37,14 +41,11 @@
   },
   "devDependencies": {
     "@glenjamin/eslint-config": "^2.1.0",
-    "@types/react": "^0.14.57",
```

Comment:
Could you leave the dev dependencies in for the latest react version?
The script which overrides these and installs the right ones will just ignore them anyway.